hlx_statics/components/code.js

```diff
@@ -1,7 +1,8 @@
 export default function decoratePreformattedCode(block) {
   const pre = block.querySelector('pre');
   // see https://prismjs.com/plugins/line-numbers/#how-to-use
-  pre.classList.add('line-numbers');
+  if (pre)
+    pre.classList.add('line-numbers');
```

Comment:
You can shorten this even more with 

```suggestion
    pre?.classList.add('line-numbers');
```